llvm/test/tools/llvm-split/AMDGPU/declarations-debug.ll

```diff
@@ -0,0 +1,12 @@
+; RUN: llvm-split -o %t %s -j 2 -mtriple amdgcn-amd-amdhsa --debug
```

Comment:
this test was missing a filecheck line and is not needed after #104763 anyway, so I removed it

Reply:
Ach, sorry about that!